src/Analyzers/CSharp/Analyzers/UseCollectionExpression/CollectionExpressionMatch.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UseCollectionExpression;
 
-/// <inheritdoc cref="Match{TStatementSyntax}"/>
+/// <inheritdoc cref="Match"/>
 internal readonly record struct CollectionExpressionMatch<TMatchNode>(
```

Comment:
note: we have a few match structs that are all about the same.  I want to unify them in the future.